app/services/school_data_importer.rb

```diff
@@ -5,16 +5,22 @@
 
 class SchoolDataImporter
   attr_reader :logger
+  attr_reader :start_year
 
-  def initialize(logger)
+  def initialize(logger, start_year = Time.zone.now.year)
     @logger = logger
+    @start_year = start_year
   end
 
   # TODO: Register and Partner 147: Figure out how exactly we want to handle school updating
   def run
     CSV.foreach(schools_data_file.path, headers: true, encoding: "ISO-8859-1:UTF-8") do |row|
       school = row_to_school(row)
-      school.save!
+      local_authority = row_to_local_authority(row)
+      local_authority_district = row_to_lad(row)
+
+      link_school_to_local_authority(school, local_authority)
+      link_school_to_lad(school, local_authority_district)
     end
```

Comment:
You know our new favourite question? 
How are we gonna test that? :P 

I guess we could set up a school with a different LAD / LA than the one in our csv file, import, and check the behaviour is as expected?